pulpcore/pulpcore/app/tasks/importer.py

```diff
@@ -51,6 +51,5 @@ def sync(importer_pk):
             new_version.delete()
             raise
 
-    if new_version.added().count() == 0 and new_version.removed().count() == 0:
-        log.debug('no changes; deleting repository version')
-        new_version.delete()
+    new_version.repository.last_version += 1
```

Comment:
It may be worth directly setting it to the value ``new_version.number``. In case there were ever a bug around these numbers getting out of sync, that would bring them back together.